serial_communication/web/app.py

```diff
@@ -3,7 +3,7 @@
 
 app = Flask(__name__)
 # Replace with your actual serial port
-ser = serial.Serial('/dev/ttyUSB0', 9600)
+ser = serial.Serial('/dev/ttyACM0', 115200)
```

Comment:
try to play with environment variables, like i want to say that create environment variables and assign them values like this:
TTGO_PORT: /dev/ttyACM0

Reply:
got it, I was thinking about it i might create a task for it.